camayoc/qcs_models.py

```diff
@@ -412,7 +413,9 @@ def __init__(
 
         # valid scan types are 'connect' and 'inspect'
         self.scan_type = scan_type
-        self.options = {'max_concurrency': max_concurrency}
+        self.options = {'max_concurrency': max_concurrency,
+                        'disabled_optional_products':
+                            disabled_optional_products}
```

Comment:
Can you confirm that no other tests are broken by this change? I've got what has over time been a kindof brittle "equivalent" method for each of the qcs objects that a lot of the tests use to assert what is on the server matches what I have locally.

Its been problematic because sometimes locally all we have are a list of  the credential ids, for example, but the server returns a list of dictionaries or something like that.